content/docs/command-reference/destroy.md

```diff
@@ -16,12 +16,14 @@ usage: dvc destroy [-h] [-q | -v] [-f]
 directory from the <abbr>project</abbr>.
 
 Note that the <abbr>cache directory</abbr> will be removed as well, unless it's
-set to an
-[external location](/doc/use-cases/shared-development-server#configure-the-external-shared-cache)
-(by default a local cache is located in `.dvc/cache`). If you were using
-[symlinks for linking](/doc/user-guide/large-dataset-optimization) data from the
-cache, DVC will replace them with the latest versions of the actual files and
-directories first, so that your data is intact after destruction.
+set to an [external location][external-cache] (by default a local cache is
+located in `.dvc/cache`). If you have setup
+[symlinks](/doc/user-guide/large-dataset-optimization) (from cache to workspace)
+in your project, DVC will replace them with the latest versions of the actual
+files and directories first, so that your data is intact after destruction.
+
+[external-cache]:
+  /doc/user-guide/managing-external-data#setting-up-an-external-cache
```

Comment:
Unrelated note: what if we start extracting long links to this format? Especially if they have `#anchors`. It can help us separate those links to manage them better (rn it's hard to detect when they break).

> Stole the idea from @iesahin 🕵️

Reply:
Need to check if the link checker supports this.

Reply:
> Need to check if the link checker supports this.

I wasn't sure how to double check this 😢. Did it seem to work?

Reply:
From https://github.com/iterative/dvc.org/pull/2491#discussion_r645739606 it sounds like this isn't supported yet ): but we should probably open an issue about that instead of limiting our MD links?